HIP-XXXX.md

```diff
@@ -0,0 +1,298 @@
+|   Other-chain proof: [                                           |
+|     Proof:<proof>                                                |
+|     Data: example.proofofconcept. DS                             |
+|   ]                                                              |
+|   DNSSEC Chain:                                                  |
+|     _443._tcp.example.proofofconcept. TLSA                       |
+|     RRSIG(_443._tcp.example.proofofconcept. TLSA)                |
+|     example.www. DNSKEY                                          |
+|     RRSIG(example.www. DNSKEY)                                   |
++------------------------------------------------------------------+
++------------------------------------------------------------------+
+| Handshake CA: proofofconcept.                                    |
++------------------------------------------------------------------+       
+| Version: 3 (0x2)                                                 |
+| Serial Number: ............                                      |  
+| X509v3 extensions:                                               |
+|   X509v3 Name constraints:                                       |
+|     proofofconcept                                               |
+|   X509v3 Basic Constraints:                                      |
+|     CA:TRUE                                                      |
+|   Urkel: [                                                       |
+|     Root: 6c3078b26c3f7858d78bd2b63530d66b413f32f0ace083e8f21... |
+|     Proof: <proof>                                               |  
+|     Data: 0x............._delegation. NS                         | 
+|   ]                                                              |
++------------------------------------------------------------------+
+```
+
+
+The following is the internal TBS structure of an X509 certificate (RFC 5280).
+
+```
+TBSCertificate  ::=  SEQUENCE  {
+        version         [0]  EXPLICIT Version DEFAULT v1,
+        serialNumber         CertificateSerialNumber,
+        signature            AlgorithmIdentifier,
+        issuer               Name,
+        validity             Validity,
+        subject              Name,
+        subjectPublicKeyInfo SubjectPublicKeyInfo,
+        issuerUniqueID  [1]  IMPLICIT UniqueIdentifier OPTIONAL,
+                             -- If present, version MUST be v2 or v3
+        subjectUniqueID [2]  IMPLICIT UniqueIdentifier OPTIONAL,
+                             -- If present, version MUST be v2 or v3
+	extensions      [3]  EXPLICIT Extensions OPTIONAL
+                             -- If present, version MUST be v3		
+```
+
+In the extensions field, we can define custom extensions. Each extension has an OID (Object identifier, a unique id) called `extnID` and the value of the extension `extnValue` is an ASN.1 DER encoded structure. An ASN.1 OCTET STRING should be suitable for storing arbitrary data.

```

Comment:
Do we need to get an official OID registered?

Reply:
Yeah an official OID would be great. Let's wait until we have a PoS and i will get one.